lib/components/viewers/stop-viewer.js

```diff
@@ -322,8 +329,8 @@ const mapStateToProps = (state, ownProps) => {
     stopData,
     stopViewerArriving: state.otp.config.language.stopViewerArriving,
     stopViewerConfig,
-    timeFormat: getTimeFormat(state.otp.config),
     transitOperators: state.otp.config.transitOperators,
+    use24HourFormat: state.user?.loggedInUser?.use24HourFormat ?? false,
```

Comment:
Is this prop still needed?

Reply:
Nope, removed in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/2ccefb263ad8863edab72e9d796596beda7336f0